pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -23,8 +23,8 @@
 // This represents a limit on the number of tail calls that can be executed within the frames_filter program.
 // The number of frames to parse is determined by HTTP2_MAX_FRAMES_ITERATIONS, resulting in a total defined as:
 // HTTP2_MAX_FRAMES_ITERATIONS * HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER (for tail call 0-1)
```

Comment:
please correct this documentation as we are not using the tail calls anymore

Reply:
I am not sure I follow. To me the documentation is correct, as we are still using a filtering tail call (just one instead of two) and the mechanism to do recursive tail calls when filtering is still present in the plain text version code (and will be added in the TLS version).

The one thing I would change is the formula for the maximum number of frames to process, to `HTTP2_MAX_FRAMES_ITERATIONS = HTTP2_MAX_FRAMES_TO_FILTER * HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER`.